relay/server/peer.go

```diff
@@ -175,25 +175,20 @@ func (p *Peer) handleHealthcheckEvents(ctx context.Context, hc *healthcheck.Send
 }
 
 func (p *Peer) handleTransportMsg(msg []byte) {
-	version, peerID, err := messages.UnmarshalTransportID(msg)
+	peerID, err := messages.UnmarshalTransportID(msg[messages.SizeOfProtoHeader:])
```

Comment:
The SizeOf* variables should be exported. Just pass the full msg []byte to the UnmarshalTransportID and handle inside the offsets in the Unmarshal functions